gef-ui/server/services/api.js

```diff
@@ -153,9 +135,9 @@ const getEligibilityCriteria = async () => {
   }
 };
 
-const getCompaniesHouseDetails = async (companyRegNumber, exporterId) => {
+const getCompaniesHouseDetails = async (companyRegNumber) => {
   try {
-    const { data } = await Axios.get(`/gef/company/${companyRegNumber}`, { params: { exporterId } });
+    const { data } = await Axios.get(`/gef/company/${companyRegNumber}`);
     return data;
   } catch (err) {
     return apiErrorHandler(err);
```

Comment:
Please add `console.error('Unable to get company house details', { err });`

Reply:
Done - there's loads of other functions in this file that could do with the same treatement, but i've added just for this